src/components/DomEvents.js

```diff
@@ -127,18 +132,20 @@ function DomEvents() {
     [setEventCount],
   );
 
-  const setPreviewRef = useCallback((node) => {
-    previewRef.current = node;
-
-    if (!node) return;
-
-    addLoggingEvents(node, (event) => {
+  useEffect(() => {
+    if (!previewRef.current) return;
```

Comment:
```suggestion
    if (!previewRef.current) {
      return;
    }
```

Reply:
I think we should add an eslint rule for this one, so it will be auto fixed on commit.

Reply:
Good point!

Reply:
I have changed the implementation